python/python/bystro/prs/preprocess_for_prs.py

```diff
@@ -219,6 +219,11 @@ def select_max_effect_per_bin(merged_scores_genos: pd.DataFrame) -> pd.DataFrame
 def clean_dosage_for_analysis(merged_scores_genos: pd.DataFrame, column_to_drop: str) -> pd.DataFrame:
     """Drop missing values and adjust scores."""
     column_to_drop_index = merged_scores_genos.columns.get_loc(column_to_drop)
+
+    # If there is more than 1 column found, our "columns_to_keep" function will not work
+    if not isinstance(column_to_drop_index, int):
```

Comment:
This was needed because get_loc doesn't actually return an int unless the column value being searched is unique. When I updated the pandas stubs, mypy began to complain about this, which is pretty neat, statically preventing errors!

Reply:
I don't believe this exists in the prs code anymore, but that is pretty neat

Reply:
it's still there, you renamed it though `format_col_index`